mc/query/query.peg.go

```diff
@@ -0,0 +1,2108 @@
+package query
```

Comment:
no don't, let's include it and track it in the tree just like the compiled protos.
There is no reason to have a peg dependency for building -- it's only required  when you are changing the grammar and want to rebuild.